docker/standalone/compose.yaml

```diff
@@ -1,15 +1,60 @@
-version: '3'
-
 services:
-  controlpanel_standalone:
-    build:
-      context: ../../
-      dockerfile: ./docker/standalone/Dockerfile
-    container_name: controlpanel_standalone
-    restart: on-failure
+  controlpanel:
+    image: ghcr.io/ctrlpanel-gg/panel:latest
+    container_name: controlpanel_panel
+    restart: unless-stopped
+    depends_on:
+      - redis
     ports:
       - "80:80"
       - "443:443"
     volumes:
-      - './website_files:/var/www/html:rw' # change it
-      - './nginx_config:/etc/nginx/conf.d/:rw' # change it
+      - './logs:/var/www/html/storage/logs:w'
+      - './env_file:/var/www/html/.env'
```

Comment:
unfortunately this will not work it need to be like the dev docker, ctrlpanel is not really made to be inside a docker and is executing some command like docker compose :( and some people have themes and addons
so you need the panel code to always be saved, there will be an update script dont worry :) 
so you need website and nginx to be saved as volume

Reply:
executing composer works fine. it installs composer just as it would for a normal install upon container start. i dont see an issue there. for themes and addons i would just need to know in which folders they would be saved and they could be exposed just like the logs.

Reply:
Yeah having composer install at each start is not very great, and most people use theme and addons which are not supported by default they change files of the panel any file

Reply:
I have exactly the same issue whit the pterodactyl panel where I need the file for my modified ptero and each time I start it it’s re uploading default configuration of eggs and it’s very annoying

Reply:
It is *not* running composer install on every container startup.
And for themes and addons, as i said, if you tell me what directories i need to be exposed, i can add a bind mount for them.

Reply:
yes since the vendor folder will not be present so it will run composer on every startup
and like I said addons etc are not officially supported and are modifying any files so we cannot know

Reply:
Docker doesnt work like that. The stuff will still be be persistent until either purging the pulled image or pulling a new version (at this point you want to re-run composer install anyways) so i dont see any issues there.
Also if Addons etc are not official supported anyways, there is even less reason to care about that case.
BUT: re-adding the entire project optionally should be still possible, i can test that real quick and re-add it.
That should be a great compromise. What do you think about that?

Reply:
addons are not supported by the code, but there is an official market full of them, they are very used
but they change any file they dont use API or things like that, its theme, new functionality etc...

Reply:
Updated it, please review.

Reply:
the current standalone will still not work whit addons and modification, adding by default the project files would be better :)

Reply:
I do not agree with that. I changed it so the project files are exposed by default. Here a couple reasons:
- You yourself told me a few days ago: the  lesser permission the better -> the normal user doesn't need access to the project files at all
- It might confuse users since they most likely wont need this anyways.
- It's a weird a bad practice

So i still think having by default the project files NOT exposed would be the much more appropriate way. And the people still got the option to safely expose them at *any time later* if they need to, without breaking anything. Exposing it by default just introduces different problems.

Reply:
like I said ctrlpanel does not support addons correctly in the code and need the project files exposed for that, many people use a modified panel
and the default nginx config will be invalid since it need to be changed
these two things need to be exposed or available for exposing
but the nginx config is really needed (for SSL and things)
and it does not concern permission they are the same if there is a volume or not

Reply:
If the nginx config of the docker container needs to be touched for addons, i guess thats alright. but it shouldnt be touched at all for any SSL/TLS stuff, that should absolutely be handled outside of the docker container by a reverse proxy and absolutely NOT by ctrlpanel itself ever!!!
if the docker compose comes with some reverse proxy, thats totally fine and legit. but the nginx inside of the docker container handling the stuff for the panel shouldn't do any of the TLS stuff!

I re-added the nginx config bind mount on your wish.